src/visitpy/visitpy/PyDatabaseCorrelation.C

```diff
@@ -320,10 +220,21 @@ DatabaseCorrelation_GetName(PyObject *self, PyObject *args)
     return retval;
 }
 
+/*static*/ PyObject *
+DatabaseCorrelation_SetNumStates(PyObject *self, PyObject *args)
+{
+    DatabaseCorrelationObject *obj = (DatabaseCorrelationObject *)self;
+
+    int ival;
+    if(!PyArg_ParseTuple(args, "i", &ival))
+        return NULL;
 
```

Comment:
May need to take a look at PyDatabaseCorrelation, it has custom code and should have been skipped by gen_python_all. Problably needs to be hand-modified.

Reply:
Alternatively, do you know of some unique string for the *custom* cases I can grep for?

Reply:
Take a look in visitpy/CMakeLists.txt:
https://github.com/visit-dav/visit/blob/6e0161b081c01771181ddeeee1a93390dadc0862/src/visitpy/CMakeLists.txt#L58-L70

https://github.com/visit-dav/visit/blob/6e0161b081c01771181ddeeee1a93390dadc0862/src/visitpy/CMakeLists.txt#L77-L82

https://github.com/visit-dav/visit/blob/6e0161b081c01771181ddeeee1a93390dadc0862/src/visitpy/CMakeLists.txt#L109-L117